src/apps/mesoscale-explorer/ui/states.tsx

```diff
@@ -337,6 +376,36 @@ export class ExampleControls extends PluginUIComponent {
     }
 }
 
+export class ColorLoaderControl extends PluginUIComponent {
```

Comment:
Can we use functional components instead now? There might be time when react deprecates class components in the future, so the fewer we have the better.

Reply:
in ui/states.tsx most are class extending PluginUIComponent, should they all be replace ? Also do you have a simple example of  functional components that can be used instead ?

Reply:
For example https://github.com/molstar/molstar/blob/master/src/mol-plugin-ui/viewport/screenshot.tsx#L164